examples/grails3-hibernate5/build.gradle

```diff
@@ -5,33 +5,33 @@ dependencies {
     implementation "org.springframework.boot:spring-boot-autoconfigure"
     implementation "org.springframework.boot:spring-boot-starter-actuator"
     implementation "org.springframework.boot:spring-boot-starter-tomcat"
```

Comment:
Are the other implementation statements even needed with the bom?

Reply:
A `bom` will not change the need to import all required dependencies.

I have not cleaned up the list of dependencies in this PR. Only removed the versions that are already provided by `grails-bom`. I am sure the dependency list can be cleaned up a bit as well.